src/routes/tokens.svelte

```diff
@@ -79,12 +79,12 @@
   bind:opened={addTokenModalOpened}
   confirmation={true}
   onConfirm={addToken}>
-  <h3 style="text-align: center;">Token Contract ID</h3>
+  <h3 style="text-align: center;">Add Token</h3>
```

Comment:
Perhaps for the future, we can make that "add token" modal into a component of its own.